docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,651 @@
+        commentID: "${{ outputs['add-progress-comment'].commentID }}"
+```
+
+### Delete Comment
+
+Removes a specific comment from a Jira issue.
+
+#### Configuration
+
+| Name                      | Type     | Required | Description                         |
+| ------------------------- | -------- | -------- | ----------------------------------- |
+| `deleteComment.issueKey`  | `string` | Y        | The Jira Issue Key (e.g., EXT-123). |
+| `deleteComment.commentID` | `string` | Y        | The ID of the comment to delete.    |
+
+#### Output
+
+This step does not produce any output values.
+
+#### Example
+
+This example deletes a specific comment from a Jira issue.
+
+```yaml
+steps:
+  - uses: jira
+    config:
+      credentials:
+        secretName: jira-credentials
+      deleteComment:
+        issueKey: "${{ freightMetadata(ctx.targetFreight.name, 'jira-issue-key') }}"
+        commentID: "${{ outputs['previous-comment-step'].commentID }}"
+```
+
+## Status Tracking
+
+### Wait for Status
+
+Waits for a Jira issue to reach a specific status before proceeding.
+
+#### Configuration
+
+| Name                           | Type     | Required | Description                                                    |
+| ------------------------------ | -------- | -------- | -------------------------------------------------------------- |
+| `waitForStatus.issueKey`       | `string` | Y        | The Jira Issue Key (e.g., EXT-123).                            |
+| `waitForStatus.expectedStatus` | `string` | Y        | The expected status to wait for (e.g., 'IN PROGRESS', 'DONE'). |
+
+#### Output
+
+This step does not produce any output values.
+
+#### Example
+
+This example waits for a change request issue to be approved before proceeding with deployment.
+
+```yaml
+steps:
+  - uses: jira
+    config:
+      credentials:
+        secretName: jira-credentials
+      waitForStatus:
+        issueKey: "${{ freightMetadata(ctx.targetFreight.name, 'change-request-key') }}"
+        expectedStatus: "Approved"
+  # Deployment steps continue after approval...
+  - uses: helm-template
+    config:
+      path: ./charts
+      vars:
+        imageTag: "${{ imageFrom(vars.imageRepo).Tag }}"
+        environment: "${{ ctx.stage.name }}"
+```
+
+## Freight Metadata Integration
+
+The Jira extension automatically stores created issue keys in the Freight metadata, allowing subsequent stages to reference the same issue. This enables tracking a single Jira issue across multiple promotion stages.
+
+### Accessing Issue Keys from Freight Metadata
+
+Use the `freightMetadata` template function to retrieve issue keys stored by previous stages:
+
+```yaml
+# Access the default issue key
+issueKey: ${{ freightMetadata(ctx.targetFreight.name, 'jira-issue-key') }}
+
+# Access a custom issue key (when issueAlias was used during creation)
+issueKey: ${{ freightMetadata(ctx.targetFreight.name, 'my-custom-alias') }}
+```
+
+## Multi-Stage Workflow Example
```

Comment:
I like this wording of "Multi-Stage Workflow" and want to suggest rewording "Freight Metadata Integration" as "Multi-Stage Workflow" and nesting this example within that section.

My rationale is that working with one issue across multiple Stages/Promotions is going to be the thing users care about and potentially go looking for information on. The metadata is just an implementation detail. imho, focusing on the use case instead of the implementation will make it easier for readers to find relevant info faster.